packages/ketcher-react/src/script/editor/tool/sgroup.ts

```diff
@@ -239,8 +243,8 @@ class SGroupTool {
       return
     }
 
-    if (!ci) {
-      //  ci.type == 'Canvas'
+    if (!closestItem) {
+      //  when closestItem.type is 'Canvas'
```

Comment:
I think this comment is not needed

Reply:
[remove redundant comment](https://github.com/epam/ketcher/pull/2007/commits/cefe470cf605c5be0b5550b6187d9af2fb031c7e)
[cefe470](https://github.com/epam/ketcher/pull/2007/commits/cefe470cf605c5be0b5550b6187d9af2fb031c7e)